src/main/java/org/fundacionjala/sfdc/pageobjects/Home/HomePageClassic.java

```diff
@@ -0,0 +1,18 @@
+package org.fundacionjala.sfdc.pageobjects.Home;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePageClassic extends HomePage {
+    @FindBy(css = "#home_Tab")
+    private WebElement homeTab;
+
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)